### PR TITLE
Terms & Conditions Protos

### PR DESCRIPTION
**Pull request by Princy2909**
protos for Terms & Conditions screen have been added by pulling from the develop branch and integrating the latest changes accordingly

### CODE REVIEW

File: protos-frontend/home/screen_options.proto [hunk 1]
```diff
@@ -8,6 +8,8 @@ import "protos-frontend/ui/LocationConsent.proto";
 import "protos-frontend/ui/Button.proto";
 import "protos-frontend/ui/CheckBox.proto";
 import "protos-frontend/ui/Modal.proto";
+import "protos-frontend/ui/Terms.proto";
```
Comment (Jha2508) [inline protos-frontend/home/screen_options.proto]: This is not need to be imported here for now

File: protos-frontend/payments/screen_options.proto [hunk 2]
```diff
@@ -8,6 +8,8 @@ import "protos-frontend/ui/LocationConsent.proto";
 import "protos-frontend/ui/Button.proto";
 import "protos-frontend/ui/CheckBox.proto";
 import "protos-frontend/ui/Modal.proto";
+import "protos-frontend/ui/Terms.proto";
```
Comment (Jha2508) [inline protos-frontend/payments/screen_options.proto]: same as above

File: protos-frontend/home/screen_options.proto (hunk 1 above)
Comment (Princy2909) [reply protos-frontend/home/screen_options.proto]: done

File: protos-frontend/payments/screen_options.proto (hunk 2 above)
Comment (Princy2909) [reply protos-frontend/payments/screen_options.proto]: done

File: protos-frontend/UI/horizontal_icon_text.proto [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * Component message
+ * Represents a ui component with a URL, image, and text.
+ */
+message HorizontalIconText {
+  string url = 1;
+  ui.Image image = 2;
+  ui.Text text = 3;
+  bool is_terms = 4;
```
Comment (Jha2508) [inline protos-frontend/UI/horizontal_icon_text.proto]: not required

**Review by Jha2508 — APPROVED** [review]

**Pull request merged**